### DISCUSSION TITLE
CTRL+ENTER in the PR GUI when typing a comment now closes the PR instead of creating a comment

### DISCUSSION THREAD

**Discussion by mijdavis2** [community/community#19701] [Pull Requests]
After the most recent service interruption, CTRL+ENTER in the PR gui when typing a comment now closes the PR instead of creating a comment.

I've closed (and reopened) tons of PRs accidentally since the service interruption which creates a ton of time waste as actions need to rebuild. Please fix the GUI so that CTRL+ENTER works the way it use to in PR comments.

**Comment by mijdavis2** [comment]
Apparently this is only happening on linux in FF so maybe it's something to do with browser compatibility or an issue with the latest FF version itself.

**Reply by MarijnS95** [reply]
It also happens for me on chrome canary on Linux, and I'm pretty sure it was fine >6 hours ago on the exact same machine.

**Reply by EmilioLaiso** [reply]
+1 for me on Chrome on Windows.

**Reply by xdumaine** [reply]
I'm experiencing it in OS X Firefox (with CMD+ENTER)

**Comment by rilysh** [comment]
@mijdavis2 For bug reports kindly raise a ticket on GitHub support https://support.github.com/contact/bug-report

Hope this works.

**Reply by xdumaine** [reply]
I opened a ticket on support, just FYI.

**Comment by xdumaine** [comment]
Might mark this as resolved/answered. I got a message from support a while back that it was indeed a bug, and has been fixed.
